### ISSUE TITLE
Incorrect URL when the source image has uppercase .JPG extension

### ISSUE THREAD

**Issue by andygott** [nystudio107/craft-imageoptimize-sharp#4]
Thanks for your work on this plugin — it’s been really useful for us on various sites since we’ve switched to using the AWS serverless image handler.

### Describe the bug
When the source image filename uses the uppercase .JPG extension, the optimised image URL uses the invalid `JPG` instead of `jpeg` as the key in the `edits` object.

```
{
  "bucket":"BUCKETNAME",
  "key":"uploads/SOME_FILE.JPG",
  "edits": {
    "JPG": { // <-- Should be `jpeg`
      "quality":100
    },
    "resize": {
      "width":1280,
      "fit":"inside"
    },
    "sharpen":true
  }
}
```

### To reproduce
Install the Craft Image Optimise plugin, set up with AWS serverless image handler, upload any image using the .JPG extension, and render the optimised image in a template.

### Expected behaviour
Optimised image is rendered.

### Versions
- Plugin version: 1.6.36
- Craft version: Craft Pro 3.7.16

I *think* a `strtolower` here might fix this?…

https://github.com/nystudio107/craft-imageoptimize-sharp/blob/645bcb9d9a1e682e8903b9c56caa480270e1a579/src/imagetransforms/SharpImageTransform.php#L118

**Comment by khalwat** [comment]
Can you try hot-applying that fix to see if it works for you? I think it might fix it on my end, but I seem to recall an issue with ServerlessSharp not handling uppercase suffixes correctly... though they may have fixed that.

**Comment by khalwat** [comment]
Addressed in 1.0.8 -> https://github.com/nystudio107/craft-imageoptimize-sharp/releases/tag/1.0.8
